task/04-arrays-tasks.js

```diff
@@ -415,7 +440,24 @@ function toStringList(arr) {
  *      { country: 'Russia',  city: 'Saint Petersburg' }
  */
 function sortCitiesArray(arr) {
-   throw new Error('Not implemented');
+   return arr.sort(sortByRule);
+
+   function sortByRule(firstElement, secondElement)
+   {
+      if (firstElement['country'] == secondElement['country'])
+      {
+         return RuleOfSort(firstElement['city'], secondElement['city']);
```

Comment:
method should contain a verb in its name

Reply:
Hi Alex! I will correct all the comments, but later.

Reply:
Hi Alex! Here is correct PR : https://github.com/AisBrestEDU/js-assignments/pull/149 . Notes fixed.